src/controllers/tripRequest.comments.js

```diff
@@ -0,0 +1,108 @@
+import db from '../database/models/index';
+import jwt from 'jsonwebtoken';
+const Comment = db['Comment'];
+const users = db['users'];
+const tripRequests = db['tripRequest'];
+
+export const commentOnRequests = async (req, res, next) => {
+  try {
+    const { comment } = req.body;
+    const userId = req.user.id;
+    const tripRequestId = req.params.tripRequestId;
+    const tripRequest = await tripRequests.findByPk(tripRequestId);
+
+    if (!tripRequest) {
+      return res.status(400).json({
+        error: 'The trip request you are trying to comment on does not exist!',
+      });
+    }
```

Comment:
this logic will allow any user to comment on your trip request, so you must make sure that trip request belongs to the current authorized user.

Reply:
> this logic will allow any user to comment on your trip request, so you must make sure that trip request belongs to the current authorized user or the comment is being given by the manager.

Can't users comment on each others requests?? @joshwambere

Reply:
i think trip-request information is confidential to the requester, so requester  and manager should only be able to comment on that trip request from my understanding

Reply:
> i think trip-request information is confidential to the requester, so requester and manager should only be able to comment on that trip request from my understanding

Okay let me do that